public/app/plugins/datasource/influxdb/query_ctrl.ts

```diff
@@ -8,6 +8,9 @@ import { TemplateSrv } from '@grafana/runtime';
 import { InfluxQuery } from './types';
 import InfluxDatasource from './datasource';
 
+const GET_MEASUREMENTS_TIMEOUT = 5 * 60 * 1000;
+const GET_MEASUREMENTS_THROTTLE_DELAY = 1000;
```

Comment:
i chose the delay arbitrarily, if anyone has a better number-suggestion, i'm open to changes